tests/capabilities.test.js

```diff
@@ -0,0 +1,237 @@
+      { timeout: 7000, interval: 100 }
+    );
+  });
+
+  afterAll(async () => {
+    await broker.stop();
+  });
+
+  describe('profile read (invite)', () => {
+    test('is present', async () => {
+      const inviteCap = await getUserInviteCap(users[0]);
+      expect(inviteCap).toBeTruthy();
+    });
+
+    test('is accessible for anonymous users by URI', async () => {
+      const inviteCap = await getUserInviteCap(users[0]);
+
+      const fetchRes = await fetch(inviteCap.id, { headers: { accept: MIME_TYPES.JSON } });
+      expect(fetchRes.ok).toBeTruthy();
+
+      const fetchedCap = await fetchRes.json();
+      expect(fetchedCap).toMatchObject(inviteCap);
+    });
+  });
+
+  describe('container access is restricted', () => {
+    test('owner can GET capability container resources', async () => {
+      // Make an authenticated fetch for user1.
+      const fetchRes = await fetch(users[0].capabilitiesUri, {
+        headers: { authorization: `Bearer ${users[0].token}` },
+      });
+      const jsonRes = await fetchRes.json();
+      const caps = jsonRes['ldp:contains'];
+      expect(arrayOf(caps)).toHaveLength(1);
+    });
+
+    test('anonymous user sees empty container', async () => {
+      const fetchRes = await fetch(users[0].capabilitiesUri);
+      const jsonRes = await fetchRes.json();
+      const caps = jsonRes['ldp:contains'];
+      expect(arrayOf(caps)).toHaveLength(0);
+    });
+
+    test('other user sees empty container', async () => {
+      // Make an authenticated fetch for user1, fetching user0's caps.
+      const fetchRes = await fetch(users[0].capabilitiesUri, {
+        headers: { authorization: `Bearer ${users[1].token}` },
+      });
+      const jsonRes = await fetchRes.json();
+      const caps = jsonRes['ldp:contains'];
+      expect(arrayOf(caps)).toHaveLength(0);
+    });
+  });
+
+  test('migration to add invite URI capabilities', async () => {
+    // For this test, we create a third user (without adding new capabilities),
+    //  to emulate the pre-migration behavior.
+    // 1. Stop capabilities service (that defines capabilities on signup).
+    await broker.destroyService('capabilities');
+
+    // 2. Create user.
+    const newUser = await signupUser(3);
+
+    // 2.1 We use this, to get the profile URI.
+    const webIdDoc = await waitForResource(
+      2000,
+      ['url'],
+      8,
+      async () =>
+        await broker.call('ldp.resource.get', {
+          resourceUri: newUser.webId,
+          accept: MIME_TYPES.JSON,
+          webId: 'system',
+        })
+    );
+    // 2.2 Since the capabilities service is disabled, no reference should be created.
+    expect(webIdDoc['apods:capabilities']).toBeUndefined();
+
+    // 3. Start capabilities service again.
+    broker.createService(CapabilitiesService, {
+      settings: { path: '/capabilities' },
+    });
+
+    // 3.1 Run migration. Will add the capabilities. Wait a bit, until the service becomes available.
+    await delay(2000);
```

Comment:
To avoid putting delay in tests, you have a `broker.waitForServices` method. https://moleculer.services/docs/0.14/broker#Methods-of-ServiceBroker